test/e2e/tests/ppom-blockaid-alert-erc20-approval.spec.js

```diff
@@ -57,13 +56,47 @@ async function mockInfura(mockServer) {
     ['eth_gasPrice'],
     ['eth_getBalance'],
     ['eth_getBlockByNumber'],
+    ['eth_createAccessList'],
```

Comment:
adding a couple of more mocks to requests that are also triggered by ppom when performing the approve ERC20 action